install/tools/ipa-ca-install

```diff
@@ -126,8 +126,11 @@ def _get_dirman_password(password=None, unattended=False):
                 validate=False)
         except KeyboardInterrupt:
             sys.exit(130)
-        if password is None:
-            sys.exit("Directory Manager password required")
+    try:
+        installutils.dm_password_validator(password)
+    except ValueError:
+        sys.exit("Directory Manager password is invalid")
```

Comment:
Please don't use sys.exit() explicitly here, either let the exception propagate up the stack to the caller or re-raise RuntimeError as you do in ipa-kra-install